src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,113 @@
+# Manipulating Windows with Tiling, Stacking & Stacking
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into adjustment mode by bringing the window into focus, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with the `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+<video autoplay loop>
+    <source src="/images/tiling-stacking-windows/adjustment-mode.webm" />
+</video>
+
+Press and hold `SUPER` + `Shift` and use the `🠐` `🠑` `🠒` `🠓` keys to resize the window. Hit `Enter` to apply sizing.
```

Comment:
Super + Shift + Arrows moves windows between workspaces, like you covered on the Workspaces page. To resize windows, you need to press Super first, then hold Shift and use the arrow keys.